dash/backend/src/modules/auth/controllers/default.controller.ts

```diff
@@ -55,6 +58,19 @@ export class DefaultController {
         return null;
     }
 
+    @Get('check-status')
```

Comment:
new GET method that checks whether the user is logged in the same way the backend's AuthGuard does